src/xenia/kernel/xboxkrnl/xboxkrnl_xconfig.cc

```diff
@@ -138,6 +138,39 @@ dword_result_t ExGetXConfigSetting_entry(word_t category, word_t setting,
 }
 DECLARE_XBOXKRNL_EXPORT1(ExGetXConfigSetting, kModules, kImplemented);
 
+dword_result_t ExSetXConfigSetting_entry(word_t category, word_t setting,
+                                         lpdword_t buffer_ptr,
+                                         dword_t buffer_size) {
+  /* Notes:
+      Handles settings the only have a single flag/value like
+     XCONFIG_USER_VIDEO_FLAGS to swap
+  */
+  XELOGI(
+      "ExSetXConfigSetting: category: {:04x}, setting: {:04x}, new Setting: "
+      "{:08x}",
+      static_cast<uint16_t>(category), static_cast<uint16_t>(setting),
+      static_cast<uint32_t>(*buffer_ptr));
+  return X_STATUS_SUCCESS;
+}
+DECLARE_XBOXKRNL_EXPORT1(ExSetXConfigSetting, kModules, kStub);
+
+dword_result_t ExReadModifyWriteXConfigSettingUlong_entry(word_t category,
+                                                          word_t setting,
+                                                          lpqword_t buffer_ptr,
```

Comment:
I doubt that buffer must be lpqword, lpdword_t or lpvoid_t is enough.
qword as buffer_size is probably not correct. 
Also I wouldn't print a buffer as buffer might be nullptr or valid buffer size can be less than 4/8 bytes

Adhoc you can change that lower x in log for upper X.